src/servers/server.ts

```diff
@@ -0,0 +1,32 @@
+import Koa from 'koa'
+import { getConfigs } from '../config/config'
+import { getRouter } from '../routers/router'
+import render from "koa-ejs"
+
+
+
+
+type GetServerParams = {
+    port?: string | number
+    onListen?: () => void
+}
+
+const getApp = () => {
+    const koa = new Koa();
+    const router = getRouter();
+
+    koa.
+        use(router.routes());

```

Comment:
accapo un po' inutile. Comunque poi prossima settimana ti facciamo vedere come montare linter e prettier in modo tale che queste cose siano gestite automaticamente

Reply:
si, all'inizio volevo incolonnare più use e mi sono dimenticato